### PR TITLE
Attempt to provide definition and usage example for `raftIndexLagThreshold`

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1572.surge.sh

Updated pages:
- [Monitor cluster endpoints for status information](https://neo4j-docs-operations-1572.surge.sh/operations-manual/5/clustering/monitoring/endpoints)

File: modules/ROOT/pages/clustering/monitoring/endpoints.adoc [hunk 1]
```diff
@@ -114,8 +115,8 @@ true* Closing connection #0
 The  status endpoint, available at `/db/<databasename>/cluster/status`, is to be used to assist with rolling upgrades.
 For more information, see link:https://neo4j.com/docs/upgrade-migration-guide/current/version-5/upgrade-minor/#_clusters[Upgrade and Migration Guide -> Clusters].
 
-Typically, it is desired to have some guarantee that a Primary is safe to shutdown for each database before removing it from a cluster.
-Counter-intuitively, that a Primary is safe to shutdown means that a majority of the *other* Primaries are healthy, caught up, and have recently heard from that database's leader.
+Typically, it is desired to have some guarantee that a primary is safe to shutdown for each database before removing it from a cluster.
+Counter-intuitively, that a primary is safe to shutdown means that a majority of the *other* primaries are healthy, caught up, and have recently heard from that database's leader.
 The status endpoints provide the following information in order to help resolve such issues.
 
 [NOTE]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/monitoring/endpoints.adoc:122]: line 124, <<clustering-transacting-via-raft-protocol, Raft>> - this cross-ref doesn't work. Do we have anything to redirect it to?

File: modules/ROOT/pages/clustering/monitoring/endpoints.adoc [hunk 2]
```diff
@@ -177,21 +178,30 @@ The following table explains how results can be compared.
...
-                                                                                                                                                                                                            If this check is run for all Primaries, the ones that match this condition can be switched off (providing other conditions are also met).
-| `hasOneLeader`                             | For any 2 Primaries (A and B), `A.leader == B.leader && leader!=null`.                                                                                             | If the leader is different then there may be a partition (alternatively, this could also occur due to bad timing).
+| `allServersAreHealthy`                     | Every primaries' status endpoint indicates `isHealthy`==`true`.                                                                                                   | To en sure the data across the entire cluster is healthy.
```
Comment (nick-giles-neo) [inline modules/ROOT/pages/clustering/monitoring/endpoints.adoc:181]: Unrelated to your change, but I happened to notice a rogue space and an out-of-place plural:
```suggestion
| `allServersAreHealthy`                     | Every primary's status endpoint indicates `isHealthy`==`true`.                                                                                                   | To ensure the data across the entire cluster is healthy.
```

File: modules/ROOT/pages/clustering/monitoring/endpoints.adoc [hunk 3]
```diff
@@ -177,21 +178,30 @@ The following table explains how results can be compared.
...
                                                                                                                                                                                                             If the leader is unknown, that means the leader messages have actually timed out.
-| `noMembersLagging`                         | For Primary A with `lastAppliedRaftIndex` = `min`, and Primary B with `lastAppliedRaftIndex` = `max`, `B.lastAppliedRaftIndex-A.lastAppliedRaftIndex<raftIndexLagThreshold`. | If there is a large difference in the applied indexes between Primaries, then it could be dangerous to switch off a Primary.
+| `noMembersLagging`                         | For primary A with `lastAppliedRaftIndex` = `min`, and primary B with `lastAppliedRaftIndex` = `max`, `B.lastAppliedRaftIndex-A.lastAppliedRaftIndex<raftIndexLagThreshold`. | If there is a large difference in the applied indexes between primaries, then it could be dangerous to switch off a primary.
 |===
 
+[NOTE]
+====
+`raftIndexLagThreshold` helps you to monitor the lag in applying Raft log entries across a cluster and set appropriate thresholds.
```
Comment (nick-giles-neo) [inline modules/ROOT/pages/clustering/monitoring/endpoints.adoc:194]: I would add something stronger to point out that `raftIndexLagThreshold` is something the user defines, not something from the system, which is what confused Umar. For example:
> You should pick a `raftIndexLagThreshold` appropriate to your particular cluster and workload. Measuring the reported lag under normal circumstances and selecting a threshold slightly above that would be a good way to select an appropriate value.

File: modules/ROOT/pages/clustering/monitoring/endpoints.adoc (hunk 1 above)
Comment (nick-giles-neo) [reply modules/ROOT/pages/clustering/monitoring/endpoints.adoc:122]: No, we don't. We got rid of the unnecessary 'advanced' sections in the move to v5. So probably just get rid of the link.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1572.surge.sh

Updated pages:
- [Monitor cluster endpoints for status information](https://neo4j-docs-operations-1572.surge.sh/operations-manual/5/clustering/monitoring/endpoints)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1572.surge.sh

Updated pages:
- [Monitor cluster endpoints for status information](https://neo4j-docs-operations-1572.surge.sh/operations-manual/5/clustering/monitoring/endpoints)

**Review by nick-giles-neo — APPROVED** [review]

**Pull request merged**